checklistbank-nub-ws/src/main/java/org/gbif/checklistbank/ws/nub/NubResource.java

```diff
@@ -148,7 +148,8 @@ public LookupUsage lookup(@RequestParam(value = "name", required = false) String
   )
   @ApiResponse(responseCode = "200", description = "Name usage suggestions found")
   @GetMapping(value = "match")
-  public NameUsageMatch match(@RequestParam(value = "name", required = false) String scientificName2,
+  public NameUsageMatch match(@RequestParam(value = "usageKey", required = false) Integer usageKey,
```

Comment:
Please also add `usageKey` to the OpenAPI documentation above (line 86).  It might also be worth adding something to the description on line 78-80.

Reply:
Done